test/apps/iodemo/run_io_demo.sh

```diff
@@ -17,6 +17,15 @@ verbose_log()
 	is_verbose && echo "$@"
 }
 
+# Show error message and exit
+error()
+{
+	echo
+	echo -e "Error: $@"
+	echo
```

Comment:
maybe call ```usage``` after printing error

Reply:
it's done for cases where the error is due to invalid usage